wiki/Configuration:-Key-Bindings.md

```diff
@@ -304,15 +305,17 @@ binds {
 }
 ```
 
-#### `screenshot-screen`, `screenshot-window`
+#### `screenshot`, screenshot-screen`,`screenshot-window`
 
 Take a screenshot of the focused screen or window respectively.
 The screenshot is both stored to the clipboard and saved to disk, according to the [`screenshot-path` option](./Configuration:-Miscellaneous.md).
 
 <sup>Since: 25.02</sup> You can disable saving to disk for a specific bind with the `write-to-disk=false` property:
+<sup>Since: ...</sup> For `screenshot` and `screenshot-screen`, you can disable showing the pointer and configure the default state of the pointer using the show-pointer property:
```

Comment:
1. Since: next release
2. Both leave this line here (adjust for screenshot-screen only) and add a similar line to the separate `screenshot` section
3. You put this line in the middle between the previous sentence ending in ":" (as in, here's an example:) and the actual example code.